UI/src/app/components/panel/panel-table/panel-table.component.html

```diff
@@ -1,54 +1,54 @@
-<table mat-table [dataSource]="columns" class="mat-elevation-z8" #table
+<table mat-table [dataSource]='columns' #table
 >
-    <ng-container matColumnDef="column_indicator">
+    <ng-container matColumnDef='column_indicator'>
         <th mat-header-cell *matHeaderCellDef></th>
-        <td mat-cell *matCellDef="let column" class="drag_indicator">
+        <td mat-cell *matCellDef='let column' class="drag_indicator">
             <mat-icon>
                 drag_indicator
             </mat-icon>
         </td>
     </ng-container>
     
-    <ng-container matColumnDef="column_name">
+    <ng-container matColumnDef='column_name'>
         <th mat-header-cell *matHeaderCellDef class="caption-item"> Item. </th>
-        <td mat-cell *matCellDef="let column"> 
+        <td mat-cell *matCellDef='let column'> 
             {{column.column_name}} 
         </td>
     </ng-container>
 
-    <ng-container matColumnDef="column_type">
+    <ng-container matColumnDef='column_type'>
         <th mat-header-cell *matHeaderCellDef> Type </th>
-        <td mat-cell *matCellDef="let column"> {{column.column_type | typeToIcon }} </td>
+        <td mat-cell *matCellDef='let column'> {{column.column_type | typeToIcon }} </td>
     </ng-container>
 
-    <ng-container matColumnDef="values">
+    <ng-container matColumnDef='values'>
         <th mat-header-cell *matHeaderCellDef> Top 10 values </th>
-        <td mat-cell *matCellDef="let column">  </td>
+        <td mat-cell *matCellDef='let column'>  </td>
     </ng-container>
 
-    <ng-container matColumnDef="comments">
+    <ng-container matColumnDef='comments'>
         <th mat-header-cell *matHeaderCellDef> Comments </th>
-        <td mat-cell *matCellDef="let column">
-            <div #anchor style="width: 40px">
+        <td mat-cell *matCellDef='let column'>
+            <div #anchor>
```

Comment:
do we need div there?

Reply:
yes, otherwise popup won't be positioned properly